dask/array/reductions.py

```diff
@@ -42,72 +79,86 @@ def reduction(x, chunk, aggregate, axis=None, keepdims=None):
         return result
 
 
+np.sum = keepdims_wrapper(np.sum)
```

Comment:
I was thinking about that. We could add the functions `_sum`, which then get wrapped.